tests/test_base_client.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import recurly
+from recurly import Resource
+from .mock_resources import MyResource, MySubResource
+from .mock_client import MockClient
+import unittest.mock as mock
+from unittest.mock import Mock, MagicMock
+
+
+def get_resource_client(success):
+    conn = MagicMock()
+    conn.request = MagicMock(return_value=None)
```

Comment:
Do we need to specify `return_value=None` here? Just curious if it's any different than the line below where we just instantiate a `MagicMock` without any args

Reply:
because i want it to follow the same interface as the real object (which returns None)